apps/wing/src/commands/test/test.ts

```diff
@@ -192,20 +192,20 @@ async function testOne(
   // determine snapshot behavior
   const snapshotMode = determineSnapshotMode(target, options);
   const shouldExecute = snapshotMode === SnapshotMode.NEVER || snapshotMode === SnapshotMode.DEPLOY;
+  const testOptions = { ...options, rootId: options.rootId ?? `Test.${nanoid(10)}` };
```

Comment:
@eladb @tsuf239 When should random test IDs be used?

my understanding so far:
- if a random ID was used every time someone ran `wing test`, then [snapshot tests](https://www.winglang.io/contributing/rfcs/2024-03-24-snapshot-tests) would be broken since they would produce different snapshots each time
- for cloud testing (currently only in our monorepo, but maybe users will want this done for them in the future), if we don't randomize the ID when running `wing test`, then it's more likely for tests to fail due to spurious resource name conflicts

Reply:
it's already in use in the cloud targets (I believe it updates the snapshots each time, from my experience- since the snapshots were invented since the randomization was there before), 
this PR is for making it behave the same way in the sim 
second point is very true :) we added this randomization to be able to run the tests in parallel

Reply:
> I believe it updates the snapshots each time
I see, I wonder if it's a latent bug?

Reply:
yes, as far as I know the snapshots are not even committed: https://github.com/winglang/wing/blob/b1d01275367b3eb2476528ec56d7b29d193492b4/examples/tests/sdk_tests/.gitignore#L1